src/Movement/Kinematics/HangprinterKinematics.cpp

```diff
@@ -275,22 +274,15 @@ bool HangprinterKinematics::CartesianToMotorSteps(const float machinePos[], cons
 													size_t numVisibleAxes, size_t numTotalAxes, int32_t motorPos[], bool isCoordinated) const noexcept
 {
 	float squaredLineLengths[HANGPRINTER_AXES];
-	squaredLineLengths[A_AXIS] = LineLengthSquared(machinePos, anchors[A_AXIS]);
```

Comment:
I would prefer three separate for-loops. It keeps separate calculations separate which is good from a math reasoning perspective. It might also be good from a processor cache/optimization perspective.

Reply:
I don't prefer that. But, as you wish.
I mean, it's a style thing we could only objectively resolve by looking at assembly code, and I don't feel like to.
Your caching hypothesis is reasonable and I'm too lazy to try to refute it.
I didn't like doing this, but done.
The bike shed color is yours.